addons/test_mail/tests/test_mail_thread_mixins.py

```diff
@@ -26,6 +27,129 @@ def test_queries_batch_mail_tracking_duration(self):
         self._test_queries_batch_duration_tracking()
 
 
+@tagged('mail_thread', 'mail_track')
+class TestMailRottingMixin(MailTrackingDurationMixinCase):
```

Comment:
Mmmh.

Looks like this test gives about as much coverage as the one in crm (same use cases), while being less readable (fake models, fake stages, records named "itemX").

Would prefer having everything in crm then, as it's probably our main usage for this new feature anyway.

(Note: the person sitting on my right at the office may not agree with this, so keep this code somewhere just in case it needs to be revived :disguised_face: ).